pkg/controller/storagecluster/topology.go

```diff
@@ -219,7 +219,7 @@ func (r *ReconcileStorageCluster) ensureNodeRacks(
 // reconcileNodeTopologyMap builds the map of all topology labels on all nodes
 // in the storage cluster
 func (r *ReconcileStorageCluster) reconcileNodeTopologyMap(sc *ocsv1.StorageCluster, reqLogger logr.Logger) error {
-	minNodes := getMinNodes(sc)
+	minNodes := getMinimumNodes(sc)
```

Comment:
Unnecessary function rename?

Reply:
This was a function that was added in a different commit of the same PR. I have updated the commits to not have this function rename.